client/container_list.go

```diff
@@ -18,6 +18,10 @@ func (cli *Client) ContainerList(ctx context.Context, options types.ContainerLis
 		query.Set("all", "1")
 	}
 
+	if options.Latest {
+		query.Set("latest", "1")
+	}
```

Comment:
I'm not sure I understand this change; could you provide more information?

The API does not have a `latest` option; https://docs.docker.com/engine/api/v1.42/#tag/Container/operation/ContainerList

Using `--latest` / `-l` on the command-line is effectively a short-hand to set `--limit=1`; https://github.com/docker/cli/blob/bfe87fd39b6b8a6bbf57e1f394238f65f99501b7/cli/command/container/list.go#L79-L81

```go
if opts.nLatest && opts.last == -1 {
	options.Limit = 1
}
```

(we could probably add a warning if both `--latest` and `--limit` / `-n` are set though)